nesta/production/orms/nih_es_config.json

```diff
@@ -20,7 +11,7 @@
                     "type": "geo_point"
                 },
                 "cost_total_project": {
-                    "type": "integer"
+                    "type": "long"
```

Comment:
Not sure about the motivations for this change, as [`integer`](https://www.elastic.co/guide/en/elasticsearch/reference/6.4/number.html), mind to explain?

Reply:
the max integer size is `2147483647`, and now because we sum the total project cost over many years, this now overflows. i.e. over time some projects receive more than $2bn

Reply:
ah ok (wow)